components/WorkoutList/WorkoutList.js

```diff
@@ -1,12 +1,23 @@
-import React from 'react'
-import useSWR from 'swr'
+import useSWR, { mutate } from 'swr'
 
 export default function WorkoutList() {
   const { data, error, isLoading } = useSWR('/api/workouts')
 
   if (isLoading) return <p>Loading...</p>
   if (error || !data) return <p>Error fetching Data</p>
 
+  async function handleDelete(id) {
+    const response = await fetch(`/api/workouts/${id}`, {
+      method: 'DELETE'
+    });
+
+    if (!response.ok) {
+      console.error('Error deleting Workout');
+    } else {
+      mutate("/api/workouts");
```

Comment:
```suggestion
      mutate();
```
Instead of destructuring mutate from swr at the top, make it directly available from the result of `useSWR('/api/workouts')`.
You don’t need to pass a path when calling `mutate()` because the hook already knows the path `('/api/workouts')` and will trigger a re-fetch for that endpoint.